cfgov/paying_for_college/models/search.py

```diff
@@ -0,0 +1,44 @@
+from paying_for_college.documents import SchoolDocument
+
+
+UNSAFE_CHARACTERS = [
+    '#', '%', ';', '^', '~', '`', '|',
+    '<', '>', '[', ']', '{', '}', '\\'
+]
+
+
+def make_safe(term):
+    for char in UNSAFE_CHARACTERS:
+        term = term.replace(char, '')
+    return term
+
+
+class SchoolSearch:
+    def __init__(self, search_term, base_query=None):
```

Comment:
It looks like theres never a time when `base_query` could be passed into this class, we should remove it and clean up the search logic.

Reply:
Agree and done. This is a great suggestion. I removed `base_query` and updated search logic.